packages/block-library/CHANGELOG.md

```diff
@@ -18,7 +22,7 @@
 
 ### Enhancement
 
--   Declare blocks as __experimental in block.json to automate syncing Gutenberg packages to WordPress ([#40655](https://github.com/WordPress/gutenberg/pull/40655)).
+-   Declare blocks as \_\_experimental in block.json to automate syncing Gutenberg packages to WordPress ([#40655](https://github.com/WordPress/gutenberg/pull/40655)).
```

Comment:
I guess that wasn't intended, right?
```suggestion
-   Declare blocks as __experimental in block.json to automate syncing Gutenberg packages to WordPress ([#40655](https://github.com/WordPress/gutenberg/pull/40655)).
```

Reply:
It's a fix applied by Prettier. In Markdown, `__` has special meaning - start of the bolded section – so this change makes sense. It also looks correct when viewing the processed file.